.github/workflows/CI.yml

```diff
@@ -112,6 +112,23 @@ jobs:
         run: |
           jsonlint -q web/src/locales/*.json
 
+  check-translation-files-out-of-sync:
+    name: Translations sync
```

Comment:
Could we call this something like `Validate Translation Files` or something else that is a little more descriptive?